src/util/util.h

```diff
@@ -173,10 +173,15 @@ namespace Util
 	void hatchingOperation(const QRectF& extent, double spacing, double offset, double rotation, T& processor)
 	{
 		// Make rotation unique
-		rotation = fmod(1.0 * rotation, M_PI);
+		rotation = fmod(1.0 * rotation, 2 * M_PI);
 		if (rotation < 0)
-			rotation = M_PI + rotation;
-		Q_ASSERT(rotation >= 0 && rotation <= M_PI);
+			rotation = 2 * M_PI + rotation;
```

Comment:
I would prefer to read
~~~
rotation += 2 * M_PI;

Reply:
Done.